application/src/main/java/org/thingsboard/mqtt/broker/config/BrokerHomePageConfig.java

```diff
@@ -158,21 +162,24 @@ private int getTlsMaxPayloadSize() {
     }
 
     private boolean isBasicAuthEnabled() {
-        String basicAuthEnabledStr = System.getenv("SECURITY_MQTT_BASIC_ENABLED");
-        if (basicAuthEnabledStr != null) {
-            return Boolean.parseBoolean(basicAuthEnabledStr);
-        } else {
-            return basicAuthEnabled;
-        }
+        return isProviderEnabled(MqttAuthProviderType.MQTT_BASIC);
     }
 
     private boolean isX509AuthEnabled() {
-        String x509AuthEnabledStr = System.getenv("SECURITY_MQTT_SSL_ENABLED");
-        if (x509AuthEnabledStr != null) {
-            return Boolean.parseBoolean(x509AuthEnabledStr);
-        } else {
-            return x509AuthEnabled;
-        }
+        return isProviderEnabled(MqttAuthProviderType.X_509);
+    }
+
+    private boolean isScramAuthEnabled() {
+        return isProviderEnabled(MqttAuthProviderType.SCRAM);
+    }
+
+    private boolean isJwtAuthEnabled() {
+        return isProviderEnabled(MqttAuthProviderType.JWT);
+    }
+
+    private Boolean isProviderEnabled(MqttAuthProviderType providerType) {
```

Comment:
Why do we need to make 4 separate DB calls for this? More providers in the future - more calls.
Isn't it better to getAll/getByPage with size 10 for example?